docs/content/about/tutorial.md

```diff
@@ -214,327 +183,169 @@ The status is used and interpreted with different meanings but in most cases it
 
 > Important: you will encounter an error the second time you execute this code because the target file will be present and status will be set to "true" instead of "false". Simply remove the file with `rm /tmp/a_file` to overcome this issue.
 
-```js
-// Dependencies
-assert = require('assert')
-fs = require('fs')
-nikita = require('nikita')
-// Touch implementation
-touch = function({config}, callback){
-  fs.stat('/tmp/a_file', function(err, stat){
-    if(err && err.code !== 'ENOENT') return callback(err)
-    if(!err) return callback(null, false)
-    fs.writeFile('/tmp/a_file', '', function(err){
-      callback(err, true)
-    })
-  })
-}
-// New Nikita session
-nikita
-// First time calling touch
-.call(touch, function(err, {status}){
-  assert.equal(status, true)
-})
-// Second time calling touch
-.call(touch, function(err, {status}){
-  assert.equal(status, false)
-})
-```
+`embed:about/tutorial/samples/idempotence.js`
 
 Note, there is an existing `nikita.file.touch` action which does just that with additional functionalities such as detecting and applying changes of ownerships and permissions.
 
 ### External actions
 
-In order to reuse our new `touch` action, we could isolate it into a separate file. The new file is called a module in Node.js terminology. Nikita `call` will accept the exported object or function. A string is interpreted as the module path and will be automatically imported. Let's create two files "./lib/touch.js" and "app.js":
+In order to reuse our new `touch` action, we could isolate it into a separate file. The new file is called a module in Node.js terminology. Nikita `call` will accept the exported object or function. Let's create two files "./lib/touch.js" and "app.js":
```

Comment:
Move back `A string is interpreted as the module path and will be automatically imported.`